website/docs/r/ecs_service.html.markdown

```diff
@@ -69,6 +69,7 @@ Load balancers support the following:
 * `target_group_arn` - (Required for ALB) The ARN of the ALB target group to associate with the service.
 * `container_name` - (Required) The name of the container to associate with the load balancer (as it appears in a container definition).
 * `container_port` - (Required) The port on the container to associate with the load balancer.
+* `launch_type` - (Optional) The launch type on which to run your service. The valid values are `EC2` and `FARGATE`. Defaults to `EC2`.
```

Comment:
While you write `defaults to EC2`, we actually do not set any defaults. Is there something I'm missing?

Reply:
The AWS API default to `API` because otherwise, it would have been a breaking change for them. Might be better not the mix AWS specifics into the Terraform documentation.

Reply:
> The AWS API default to `API`

Not sure to get it here: Is `API` a typo or really a valid value? can not find anything on that.

Also, we already have default values in a lot of places, whether those are booleans, floats, etc, in order to mimic the AWS API correctly.

Reply:
It is a typo, I meant `EC2`. So you would suggest adopting the AWS default into the Schema? What about the `requires_compatibilities` field of r/aws_ecs_task_definition, it defaults to `["EC2"]`